server-spi-private/src/main/java/org/keycloak/broker/provider/AbstractIdentityProvider.java

```diff
@@ -152,11 +152,6 @@ public void preprocessFederatedIdentity(KeycloakSession session, RealmModel real
 
     }
 
-    @Override
-    public void importNewUser(KeycloakSession session, RealmModel realm, UserModel user, BrokeredIdentityContext context) {
```

Comment:
Removed this as it was never implemented - so was just confusing that this was being called